test/fuzz/test_runner.py

```diff
@@ -321,15 +334,31 @@ def job(t, args):
                 text=True,
             ).stderr
             logging.debug(output)
+            return t, output
 
         jobs.append(fuzz_pool.submit(job, t, args))
 
     for future in as_completed(jobs):
-        future.result()
+        target, output = future.result()
+        if using_libfuzzer:
+            done_stat = [l for l in output.splitlines() if "DONE" in l]
+            assert len(done_stat) == 1
+            remaining_targets = log_fuzz_progress(remaining_targets, test_list, done_stat, target)
+
+def log_fuzz_progress(remaining_targets, test_list, done_stat, target):
```

Comment:
Was there a reason you prefer to use list arguments rather than integers that store the counts? Doesn't really matter since this is just a fuzz test runner, but seems likes an unnecessary use of lists.

Reply:
no there is not, pushed [c54df63](https://github.com/bitcoin/bitcoin/pull/32354/commits/c54df63f7d5047bf8efc345f102942cbb86fb019)